src/gui/base/Dialog.ts

```diff
@@ -1006,6 +1007,30 @@ export class Dialog implements ModalComponent {
 		return dialog
 	}
 
+	/**
+	 * Show a dialog with a preview of a given list of contacts
+	 * @param contacts The contact list to be previewed
+	 * @param okAction The action to be executed when the user press the import button
+	 */
+	static importVCardDialog(contacts: Contact[], okAction: (dialog: Dialog) => unknown): Dialog {
```

Comment:
I'm sorry but this does not belong here. Dialog class if for all kinds of generic dialogs, this is a very very specific one.

Reply:
I see this Dialog as generic as the ```showRequestPasswordDialog```, but I moved it to implement the List.

Reply:
Yeah that one should also not be here to be honest :sweat_smile: